library/src/main/java/com/google/android/exoplayer/extractor/webm/WebmExtractor.java

```diff
@@ -1437,6 +1470,96 @@ public void initializeOutput(ExtractorOutput output, int trackId, long durationU
       }
     }
 
+    private static class WAVEFORMATEX {
```

Comment:
This is a huge amount of code for something that basically checks a few values. Can we just do the few-lines version?